### PR TITLE
[shopsys] moving features from project-base to the packages

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Moving features from project-base to packages is essential for the future of the platform development and upgrading abilities
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://rv-easy-movements.odin.shopsys.cloud
  - https://cz.rv-easy-movements.odin.shopsys.cloud

### CODE REVIEW

File: packages/frontend-api/src/Resources/config/graphql-types/ContactInputDecorator.types.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                    -   NotBlank:
+                            message: "Please enter your email address"
+                    -   Email:
+                            message: "Please enter valid email"
+                    -   Length:
+                            max: 255
+                            maxMessage: "Email cannot be longer than {{ limit }} characters"
+            message:
+                type: "String!"
+                description: "Message sent to recipient"
```
Comment (TomasLudvik) [inline packages/frontend-api/src/Resources/config/graphql-types/ContactInputDecorator.types.yaml]: This is past time, I think it should be a future time or time net dependant format :)

File: packages/frontend-api/src/Resources/config/graphql-types/MutationDecorator.types.yaml [hunk 2]
```diff
@@ -162,3 +162,11 @@ MutationDecorator:
                         type: RemovePromoCodeFromCartInput!
                         validation: cascade
                 resolve: "@=mutation('removePromoCodeFromCartMutation', args, validator)"
+            Contact:
```
Comment (TomasLudvik) [inline packages/frontend-api/src/Resources/config/graphql-types/MutationDecorator.types.yaml]: ContactForm seems to be more clear to me

File: packages/framework/src/Migrations/Version20240610181711.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        if ($this->isAppMigrationNotInstalledRemoveIfExists('Version20200428092429')) {
+            $this->sql('ALTER TABLE orders ALTER first_name DROP NOT NULL');
+            $this->sql('ALTER TABLE orders ALTER last_name DROP NOT NULL');
+            $this->sql('ALTER TABLE orders ALTER delivery_first_name DROP NOT NULL');
+            $this->sql('ALTER TABLE orders ALTER delivery_last_name DROP NOT NULL');
+        }
+
+        if ($this->isAppMigrationNotInstalledRemoveIfExists('Version20200622122343')) {
+            $this->sql('ALTER TABLE orders ADD gtm_coupon VARCHAR(64) DEFAULT NULL');
```
Comment (TomasLudvik) [inline packages/framework/src/Migrations/Version20240610181711.php:25]: What is a GTM coupon? I think it should be named differently

File: packages/frontend-api/src/Resources/config/graphql-types/PriceInterfaceDecorator.types.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+PriceInterfaceDecorator:
```
Comment (TomasLudvik) [inline packages/frontend-api/src/Resources/config/graphql-types/PriceInterfaceDecorator.types.yaml]: Why do we need this? I see only one usage of the interface

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: packages/frontend-api/src/Resources/config/graphql-types/ContactInputDecorator.types.yaml (hunk 1 above)
Comment (vitek-rostislav) [reply packages/frontend-api/src/Resources/config/graphql-types/ContactInputDecorator.types.yaml]: improved in 93897de5b8

File: packages/frontend-api/src/Resources/config/graphql-types/MutationDecorator.types.yaml (hunk 2 above)
Comment (vitek-rostislav) [reply packages/frontend-api/src/Resources/config/graphql-types/MutationDecorator.types.yaml]: your wish is my command f9a44cbe49

File: packages/framework/src/Migrations/Version20240610181711.php (hunk 3 above)
Comment (vitek-rostislav) [reply packages/framework/src/Migrations/Version20240610181711.php:25]: 7d893a85c9

File: packages/frontend-api/src/Resources/config/graphql-types/PriceInterfaceDecorator.types.yaml (hunk 4 above)
Comment (vitek-rostislav) [reply packages/frontend-api/src/Resources/config/graphql-types/PriceInterfaceDecorator.types.yaml]: removed in aa6097c367

**Review by TomasLudvik — APPROVED** [review]

**Pull request merged**